litiengine/src/test/java/de/gurkenlabs/litiengine/util/ColorHelperTests.java

```diff
@@ -40,44 +41,34 @@ private static Stream<Arguments> getColorFromHexString() {
         Arguments.of("#0000ff", Color.BLUE));
   }
 
-  @Test
-  void testRedFromAlphaHexString() {
-    String red200 = "#c8ff0000";
-
-    Color redDecoded = ColorHelper.decode(red200);
-
-    Color alphaRed = new Color(255, 0, 0, 200);
-    assertEquals(alphaRed.getRed(), redDecoded.getRed());
-  }
-
-  @Test
-  void testGreenFromAlphaHexString() {
-    String red200 = "#c8ff0000";
+  @Nested
+  class DecodeTest
+  {
+      String red200 = "#c8ff0000";
 
-    Color redDecoded = ColorHelper.decode(red200);
+      Color redDecoded = ColorUtil.decode(red200);
 
-    Color alphaRed = new Color(255, 0, 0, 200);
-    assertEquals(alphaRed.getGreen(), redDecoded.getGreen());
-  }
-
-  @Test
-  void testBlueFromAlphaHexString() {
-    String red200 = "#c8ff0000";
+      Color alphaRed = new Color(255, 0, 0, 200);
```

Comment:
We can use hex literals here for readability. This makes it easier to understand which parameter goes to corresponds with which byte in the `red200` variable.

EX: `new Color(0xff, 0xdd, 0xee, 0xc8);`

https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-HexIntegerLiteral